File: Remove_element.java
public class Remove_element {
    public int removeElement(int[] nums, int val) {
        if(nums == null || nums.length == 0) {
            return 0;
        }

        int i = 0; 
        int j = 0; 

        while(j < nums.length) {
            while(j < nums.length && nums[j] == val) {
                j++;
            }

            
            if(j < nums.length) {
                nums[i] = nums[j];

                i++;
                j++;
            }
        }

        return i;

    }
}

File: removeElement.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        
        n=len(nums)
        k=0
        i=0
        j=n-1
        
        while i<=j:
            if nums[i]==val:
                nums[i]=nums[j]
                j-=1
            else:
                i+=1
                k+=1
        return k
